### DISCUSSION TITLE
Bounding Box display

### DISCUSSION THREAD

**Discussion by Brayan532** [ultralytics/ultralytics#16068] [Q&A]
implementation of yolov8 in Keras leads to getting the raw data including the bounding boxes. e.g. boxes = [
    [1.314, 3.2655, 4.061, 35.858, 1.00],  # box with confidence
    [3.2099, 19.822, 4.0029, 48.796, 1.00],
    [3.5606, 37.578, 4.0002, 53.052, 0.95],
    [3.6445, 421.08, 4, 436.6, 0.8],  # lower confidence
    [3.6519, 428.88, 4, 445.43, 0.75]
]
question: How can we draw this box? I've tried the following approach but didn't work:  xmin = int(x_center - width / 2)
    ymin = int(y_center - height / 2)
    xmax = int(x_center + width / 2)
    ymax = int(y_center + height / 2)
    
    # Define top-left and bottom-right points for the rectangle
    top_left = (xmin, ymin)
    bottom_right = (xmax, ymax)

I'm not sure whether I am calculating right or not?

**Comment by glenn-jocher** [comment]
@Brayan532 to draw bounding boxes, ensure your coordinates are correct. The format you provided seems to be `[x_center, y_center, width, height, confidence]`. Your calculations for `xmin`, `ymin`, `xmax`, and `ymax` look correct. Verify that your drawing function uses these points accurately. If the issue persists, check if the image dimensions and scaling are consistent.
